include/argon2.h

```diff
@@ -13,6 +13,16 @@
 #ifndef ARGON2_H
 #define ARGON2_H
 
+#ifdef WIN32
+#ifdef argon2_EXPORTS
+#define ARGON2_API __declspec(dllexport)
+#else
+#define ARGON2_API __declspec(dllimport)
+#endif
+#else
+#define ARGON2_API
+#endif // WIN32
```

Comment:
There is already some dllexport mechanism defined a few lines below - perhaps you could reuse/improve that?

Reply:
(FYI, it is inherited from the original Argon2 code, I am not familiar with the peculiarities of building libraries on Windows...)